graylog2-server/src/main/java/org/graylog2/rest/resources/streams/alerts/StreamAlertResource.java

```diff
@@ -159,7 +159,7 @@ public AlertListSummary listPaginated(@ApiParam(name = "streamId", value = "The
         checkPermission(RestPermissions.STREAMS_READ, streamId);
 
         if (limit == 0) {
```

Comment:
This whole block could be gotten rid of by using [`@DefaultParam("300")`](https://docs.oracle.com/javaee/7/api/javax/ws/rs/DefaultValue.html) for the method parameter `int limit`.